cli/module_loader.rs

```diff
@@ -151,6 +151,12 @@ impl ModuleLoadPreparer {
     lib: TsTypeLib,
     permissions: PermissionsContainer,
   ) -> Result<(), AnyError> {
+    if roots.iter().all(|s| graph.get(s).is_some()) {
```

Comment:
Can't do this exactly because we need to do stuff like evaluate if this is a remote module importing a local one.